packages/yourloops/test/integration/assert/trends.ts

```diff
@@ -132,3 +132,54 @@ export const checkMedian = async () => {
   })
   expect(screen.getAllByTestId('cbgMedian-median')).toHaveLength(1)
 }
+
+export const checkReadings100 = async () => {
```

Comment:
Maybe you could reuse/refactor the method `checkRangeSelection()` above, given that it tests the same part of the trends and right now, both logics seem very uncoupled